### PR TITLE
Adding stoatydive recipe

### PR DESCRIPTION
**Pull request by heylf**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by heylf** [comment]
Tests have passed
@bioconda/core

**Comment by epruesse** [comment]
One of the two files comprising the package and copied to `/bin` is a module, not an entry point.

**Comment by heylf** [comment]
Tests have passed. Project folder structure has been standardized. 

@bioconda/core

File: recipes/stoatydive/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+requirements:
+    host:
+        - python
+    run:
+        - python
+        - bedtools >=2.27.1
+        - numpy
+        - matplotlib
+        - scipy =1.2
```
Comment (dpryan79) [inline recipes/stoatydive/meta.yaml]: Is scipy 1.3 not supported for some reason?

Comment (heylf) [reply recipes/stoatydive/meta.yaml]: Thanks for the remark. I will make a fix than it should work with higher versions of scipy.

Comment (heylf) [reply recipes/stoatydive/meta.yaml]: Its fixed!

**Comment by dpryan79** [comment]
Thanks @heylf !

**Comment by dpryan79** [comment]
@bioconda-bot recheck

**Pull request merged**